src/fprime/fbuild/cli.py

```diff
@@ -143,9 +143,8 @@ def add_target_parser(
         parser.add_argument(
             "-j",
             "--jobs",
-            default=1,
```

Comment:
default=None?  or is this implicit?

Reply:
It's implicit - `default`'s default value is None